dist/transport.go

```diff
@@ -0,0 +1,186 @@
+package dist
```

Comment:
Because naming has extremely important impact on security, and because it’s so easy to miss cases when parsing ambiguous strings, we typically require close to 100% unit test code coverage for the `ImageTransport` and `ImageReference` implementations.